README.md

```diff
@@ -125,6 +125,107 @@ function @ 0x4011C0
 ...
 ```
 
+Additionally, capa also supports analyzing sandbox reports for identifying capabilities
+from dynamic execution traces. This is especially relevant for samples that are either packed or
+obfuscated, since static analysis on those samples would likely not yield the desired information.
+
+Currently, capa only supports reports from the [CAPE](https://github.com/kevoreilly/CAPEv2) sandbox. 
+Here's an example run on sample 05be49819139a3fdcdbddbdefd298398779521f3d68daa25275cc77508e42310 which is packed:
+
+```yaml
+$ capa 05be49819139a3fdcdbddbdefd298398779521f3d68daa25275cc77508e42310.exe
```

Comment:
I'm not sure we need to show this here (in full) although it nicely illustrates the difference between EXE (static) / JSON (dynamic).

Reply:
maybe omit some of the capabilities, and then use a generic name _packed\_sample.{exe,json}_ instead? also specify the links to that sample's report and executable?

Reply:
yes, I think using a generic sample is fair
we could just add that the JSON report must be obtained from a CAPE instance